helm-chart/values.yaml

```
@@ -0,0 +1,79 @@
+image:
+  repository: alpacamarkets/marketstore
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart version.
+  # tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+mktsConfig: |-
+  root_directory: /data
+  listen_port: 5993
+
```

```suggestion
    timezone: UTC
    disable_variable_compression: true
    
```
I would like to add 2 more default configs

---

thanks, will add it today!